nuspec/Cake.Powershell.nuspec

```diff
@@ -15,14 +15,29 @@
         <copyright>Copyright (c) Phillip Sharpe 2015</copyright>
         <releaseNotes>Initial Release</releaseNotes>
         <tags>Cake Script Build Powershell</tags>
+		<dependencies>
+		  <group targetFramework=".NETFramework4.6">
+			<dependency id="Microsoft.PowerShell.5.ReferenceAssemblies" version="1.1.0" />
+		  </group>
+		  <group targetFramework=".NETCoreApp3.1">
+			<dependency id="Microsoft.PowerShell.SDK" version="7.0.5" />
+		  </group>
+		  <group targetFramework=".NETCoreApp5.0">
```

Comment:
Wouldn't it be better to target `netstandard2.0` over `.NETCoreApp3.1`?

Reply:
I could not compile netstandard with any of dependencies - anyway if you have some idea please share i will modify pr

Reply:
Looking at nuget.org I think we could change `.NETCoreApp5.0` to `net5.0` as [Microsoft.PowerShell.SDK v7.1.2](https://www.nuget.org/packages/Microsoft.PowerShell.SDK/7.1.2) has `net5.0` dependencies but your right I don't think there's anything we can do with `.NETCoreApp3.1` as [Microsoft.PowerShell.SDK v7.0.5](https://www.nuget.org/packages/Microsoft.PowerShell.SDK/7.0.5) only has `.NETCoreApp3.1` dependencies and the latest preview package jumps straight to `net6.0`.

The test project would probably still need to be `.NETCoreApp5.0` but in theory, the addin could target `net5.0`. I haven't checked if this would compile though, so I happy to be proven wrong.

Reply:
Hmm I used nuget package explorer - and for example net5.0 is saved as .NETCoreApp5.0. Could you please advise what could I change? test project is also targeting 3 frameworks
![nuget01](https://user-images.githubusercontent.com/1914201/108860889-f0599100-75ee-11eb-980c-3cdc2771f38b.PNG)
![nuget02](https://user-images.githubusercontent.com/1914201/108860907-f5b6db80-75ee-11eb-9cc9-3823d903d95e.PNG)
![nuget03](https://user-images.githubusercontent.com/1914201/108860924-f9e2f900-75ee-11eb-81ab-2efd82c46560.PNG)